Articles/Defining and Using Observables.md

```diff
@@ -0,0 +1,341 @@
+Observable definitions can be overridden in extensions (via `extends`) in event class and object definitions, including the special "patch" type of extends. This is also mentioned above, though repeated here to emphasize that this is a general rule for all types of observable definitions.
+
+#### Definition Nuance: Hidden Types Are Not Supported
+Defining observables in definitions of hidden event classes or object is not supported as this would lead to colliding observables `type_id` values. See [Appendix 2: Hidden Types](#appendix-2-hidden-types) for more information. This is also mentioned above, though repeated here to emphasize that this is a general rule for all types of observable definitions.
+
+#### Definition Nuance: Profiles
+Profile definitions can override attribute observables (as with any other attribute field), though cannot modify object or class-specific attribute path observables (the top-level `observable` definitions). In other words, with regard to observables profiles can only affect observables by attributes.
+
+> [!NOTE]
+> Implementation-wise, this restriction may not be difficult to overcome, however it does become difficult to conceptualize and visualize the effect of a profile if it can affect an a top level property of event class or object this way. Constraints, another top-level event class / object concept, are similarly not controllable by profiles.
+
+#### Definition Nuance: New Object Observables Discouraged
+The OCSF community currently discourages defining objects as observables. The object observable merely indicates the presence of the object in the event on a specific path. A second query would be needed to interrogate the object. It is not terribly useful.
+
+#### Definition Nuance: Observable Values Are Strings
+The `observable` object's `value` field is used for attributes that are primitive types (strings, numbers, booleans), subtypes of primitive types, and arrays of primitive types or primitive subtypes. The type of the `observable` object's `value` field is always of type `string_t`, and so string conversion may be needed. The `value` field is not populated for objects or arrays of objects.
+
+For more, see [Populating The Value Field](#populating-the-value-field).
+
+#### Definition Nuance: Event Class Observables Are Not Supported
+Defining an event class as an observable is not supported. This would be essentially redundant with the `type_uid` field, which already uniquely identifies an OCSF event's event class. In other words, a query for an event class's observable ID might as well query for event classes `type_uid`. Further, the OCSF community is trying to move away from observables by object, which this would be similar to, since these do no populate the `observable` object's `value` field.
+
+The only thing this sort of definition would enable would be to detect events that are in event class inheritance subtree at a finer grain than categories; a fairly esoteric use-case.
```

Comment:
Grammar:  "in event class inheritance subtree...": '...in an event class inheritance subtree...'?